src/lib_ccx/params.c

```diff
@@ -953,6 +960,21 @@ void version(char *location) {
 	mprint("	Git commit: %s\n", GIT_COMMIT);
 	mprint("	Compilation date: %s\n", COMPILE_DATE);
 	mprint("	File SHA256: %s\n", hash);
+
+	mprint("Libraries used by CCExtractor\n");
+	mprint("	Tesseract Version: %s\n", (const char*)TessVersion());
```

Comment:
Did you take into account a version that has no Tesseract/Leptonica support included?

Reply:
Sorry, by 'a version' you mean of CCExtractor which has no Tesseract/Leptonica support?

Reply:
Yes, it's possible to compile CCExtractor without OCR support.

Reply:
Oh I'll wrap the two statements in #ifdef then.

Reply:
@canihavesomecoffee  ping